The gps-tools package has libraries that are exported and others
that are installed without being exported.  The CMakeLists.txt file
contains an explicitly call to ament_environment_hooks to ensure                                                                                                                                                 that library_path.dsv is created.  However, this is redundant and
results in multiple commands to generate the file.  This produces
the following error during the configure task:
CMake Error: Files to be generated by multiple different commands: \
  "TOPDIR/tmp-glibc/work/core2-64-oe-linux/gps-tools/1.0.4-4-r0/build/ament_cmake_environment_hooks/library_path.dsv"
                                                                                                                                                                                                                 Signed-off-by: Rob Woolley <rob.woolley@windriver.com>

Index: git/CMakeLists.txt
===================================================================
--- git.orig/CMakeLists.txt
+++ git/CMakeLists.txt
@@ -48,10 +48,10 @@ ament_target_dependencies(utm_odometry_t
   "sensor_msgs"
 )
 
-if(NOT WIN32)
-  ament_environment_hooks(
-    "${ament_cmake_package_templates_ENVIRONMENT_HOOK_LIBRARY_PATH}")
-endif()
+#if(NOT WIN32)
+#  ament_environment_hooks(
+#    "${ament_cmake_package_templates_ENVIRONMENT_HOOK_LIBRARY_PATH}")
+#endif()
 
 install(TARGETS
     utm_odometry_component
